client/lib/post-normalizer/index.js

```diff
@@ -1,13 +1,15 @@
 /** @format */
+
 /**
  * External Dependencies
  */
-import async from 'async';
 import debugFactory from 'debug';
 const debug = debugFactory( 'calypso:post-normalizer' );
+
 /**
  * Internal dependencies
  */
+import { promisify } from '../../utils';
```

Comment:
thanks for noticing: i'll fix

Reply:
Meh. Updating it to `client/utils` didn't work. Webpack does not have the root of the package as a resolution path -- it always looks in either client or `node_modules`.  That means it was look int `client/client/utils` and `node_modules/client/utils` which both don't work.

The reason I can't just write `'utils'` is because the dependency conflict with the node module for `utils` is more trouble than its worth.  One thing that may work is writing:
`import { promisify } from 'utils/`, but even that is strange and I rather use relative paths.